solidity/ecdsa/tasks/increase-authorization.ts

```diff
@@ -0,0 +1,75 @@
+import { task, types } from "hardhat/config"
```

Comment:
Let's remove all the changes to hardhat tasks as https://github.com/keep-network/keep-core/pull/3123 handles them already.